timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/AsyncTimelockServiceIntegrationTest.java

```diff
@@ -297,18 +299,35 @@ public void testCurrentTimeMillis() throws InterruptedException {
     }
 
     @Test
-    public void lockRequestsAreIdempotent() {
-        LockToken token = cluster.lock(requestFor(LOCK_A)).getToken();
+    public void lockRequestsToRpcClientAreIdempotent() {
+        IdentifiedLockRequest request = IdentifiedLockRequest.from(requestFor(LOCK_A));
 
-        LockRequest request = requestFor(LOCK_A);
-        CompletableFuture<LockResponse> response = cluster.lockAsync(request);
-        CompletableFuture<LockResponse> duplicateResponse = cluster.lockAsync(request);
+        LockResponseV2 response = cluster.timelockRpcClient().lock(request);
```

Comment:
the previous test was more interesting for our purposes; one tests whether if i lock something that's already locked that it hands back the same token, the other makes sure that when a _different_ holder of the lock unlocks it, that both requestors acquire successfully. Basically, I think it's a less obvious constraint to enforce, seems like an obvious bug to accidentally add, and so I prefer the old test